pkgs/applications/blockchains/teos/default.nix

```diff
@@ -28,7 +29,7 @@ let
     platforms = platforms.unix;
   };
 
-  cargoPatches = [ ./add-cargo-lock.patch ];
+  cargoPatches = [];
```

Comment:
We can drop this line completely.

(and drop `cargoPatches` from `inherit` two times below)